kit/azure/bootstrap/main.tf

```diff
@@ -24,3 +24,11 @@ resource "azurerm_role_assignment" "tfstates_engineers" {
   principal_id         = azuread_group.platform_engineers.object_id
   scope                = module.terraform_state[0].container_id
 }
+
+resource "azurerm_role_assignment" "tfstates_engineers_users" {
+  for_each = toset(data.azuread_users.platform_engineers_members.object_ids)
```

Comment:
we should probably add a comment on why we do this individual assignment (in addition to the group?) that explains the whole reasoning re. eventual consistency causing problems?

Reply:
I have later a meeting with MJ, perhaps we can solve it without it.